packages/volto/src/components/manage/Form/save.css

```diff
@@ -0,0 +1,25 @@
+.sidebar-metadata-container .save-button-wrapper {
```

Comment:
please, move styles under themes folder in correct folder for this type of styles. No use .css extension

https://github.com/plone/volto/tree/main/packages/volto/theme/themes